### PR TITLE
formatting changes

### PR DESCRIPTION
**Pull request by tiffanynolan**

### CODE REVIEW

File: walkthroughs/3-low-code-api-development/walkthrough.adoc [hunk 1]
```diff
@@ -35,12 +33,15 @@ It also allows you to identify important design decisions prior to coding and de
 .Red Hat OpenShift
 ****
 * link:{openshift-console-url}[Console, window="_blank"]
+* link:https://help.openshift.com/[OpenShift Online Help Center, window="_blank"]
```
Comment (evanshortiss) [inline walkthroughs/3-low-code-api-development/walkthrough.adoc:36]: Might need to reconsider this. It links to OpenShift Online help. I can change to "https://docs.openshift.com/dedicated/3/welcome/index.html" in my PR later

Comment (evanshortiss) [reply walkthroughs/3-low-code-api-development/walkthrough.adoc:36]: (OpenShift Dedicated docs)

File: walkthroughs/3-low-code-api-development/walkthrough.adoc [hunk 2]
```diff
@@ -35,12 +33,15 @@ It also allows you to identify important design decisions prior to coding and de
...
+* link:https://blog.openshift.com/[OpenShift Blog, window="_blank"]
 ****
 
 [type=walkthroughResource,serviceName=fuse]
 .Red Hat Fuse Online
 ****
-* link:{fuse-url}[Console, window="_blank"]
+* link:{fuse-url}[Console, window="_blank", id="resources-fuse-url"]
+* link:{fuse-documentation-url}[Fuse Documentation, window="_blank"]
```
Comment (evanshortiss) [inline walkthroughs/3-low-code-api-development/walkthrough.adoc:44]: This `fuse-documentation-url` doesn't appear to be defined. Again, I can address it later.

File: walkthroughs/5-adding-data-sync-graphql/walkthrough.adoc [hunk 3]
```diff
@@ -54,8 +56,10 @@ image::images/arch.png[architecture, role="integr8ly-img-responsive"]
 [time=10]
 == Deploying the Data Sync showcase application
 
+//Home > Search > Select Templates from the drop down > Search for datasync and they should show up. Just make sure you're on the openshift namespace
```
Comment (evanshortiss) [inline walkthroughs/5-adding-data-sync-graphql/walkthrough.adoc:59]: Should these `//` chars be left here?

**Review by evanshortiss — APPROVED** [review]
Some things I'm not 100% clear on, but I can test and fix in my PR later.

File: walkthroughs/3-low-code-api-development/walkthrough.adoc (hunk 1 above)
Comment (tiffanynolan) [reply walkthroughs/3-low-code-api-development/walkthrough.adoc:36]: Yep, agreed that we should look into using a better URL here. For now, I have all the SPs using the same exact resource links for OpenShift. We should update all of them once we know what this should really be.

File: walkthroughs/3-low-code-api-development/walkthrough.adoc (hunk 2 above)
Comment (tiffanynolan) [reply walkthroughs/3-low-code-api-development/walkthrough.adoc:44]: Ok, it's possible I didn't carry over an attribute for this. The idea here is the same as for OpenShift resources. If a SP includes Fuse, it should be consistent with what the resources are and where they link to.

File: walkthroughs/5-adding-data-sync-graphql/walkthrough.adoc (hunk 3 above)
Comment (tiffanynolan) [reply walkthroughs/5-adding-data-sync-graphql/walkthrough.adoc:59]: Yes, I made this as a comment for now based on some info from Jameel. Need to validate how this actually works in OS4 with the templates once we get the uxd cluster updated to the latest changes from the past sprint.

**Pull request merged**